doc/source/reference/commands/thread_dump.rst

```diff
@@ -0,0 +1,99 @@
+.. -*- rst -*-
+
+.. groonga-command
+.. database: commands_thread_dump
+
+.. groonga-command
+.. thread_dump
+
+``thread_dump``
+===============
+
+Summary
+-------
+
+.. versionadded:: 11.1.1
+
+.. note::
+
+   Currently, this command works only on Windows.
+
+``thread_dump`` has the following a features:
+
+  * ``thread_dump`` puts a backtrace of all threads into a log as logs of
+    INFO level at the time of running this command as below.
```

Comment:
NOTICE

Reply:
Oh...
Thank you for your comment. I've fixed it.